src/views/pages/interviewees/Home.js

```diff
@@ -0,0 +1,44 @@
+// --------------------------------
+//  Define Data Sources
+// --------------------------------
+
+let getInterviewersList = async() => {
+    const options = {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json'
+        }
+    };
+    try {
+        const response = await fetch(`http://localhost:3000/interviewees.json`, options)
```

Comment:
Should have fetched the data from the controllers itself, as in this case now the data won't be updated for the interviews.